Gist/Managers/MessageManager.swift

```diff
@@ -99,6 +100,8 @@ class MessageManager: EngineWebDelegate {
                     self.showNewMessage(url: url)
                 } else {
                     dismissMessage {
+                        // BERNARD: not sure about this one
+                        self.delegate?.removePersistentMessage(message: self.currentMessage)
```

Comment:
@BernardGatt I believe this is an attempt to load another page. In this case I do not think we want to remove the persistent message so that it shows up again when he returns. Is my thinking correct here?